test/test_cases/run_solid_body_rotation.jl

```diff
@@ -42,11 +43,33 @@ function run_solid_body_rotation(
         # test successful integration
         @test step!(simulation) isa Nothing # either error or integration runs
     elseif mode == :regression
-        # TODO!: Implement meaningful(!) regression test
+        simulation = Simulation(model, stepper, dt = dt, tspan = (0.0, dt))
+        @unpack ρ, uh, w, ρe_tot = init_solid_body_rotation(FT, params)
+        set!(simulation, ρ = ρ, uh = uh, w = w, ρe_tot = ρe_tot)
+        step!(simulation)
+        u = simulation.integrator.u.nhm
+        uh_phy = Geometry.transform.(Ref(Geometry.UVAxis()), u.uh)
+        w_phy = Geometry.transform.(Ref(Geometry.WAxis()), u.w)
+
+        # perform regression check
+        current_uh_max = 3.671239932601727e-15
+        current_w_max = 0.2107507579635938
+
+        @test (abs.(uh_phy |> parent) |> maximum) ≈ current_uh_max rtol = 1e-3
+        @test (abs.(w_phy |> parent) |> maximum) ≈ current_w_max rtol = 1e-3
     elseif mode == :validation
-        # TODO!: Implement the rest plots and analyses
-        # 1. sort out saveat kwarg for Simulation
-        # 2. create animation for a rising bubble; timeseries of total energy
+        simulation = Simulation(model, stepper, dt = dt, tspan = (0.0, 3600))
+        @unpack ρ, uh, w, ρe_tot = init_solid_body_rotation(FT, params)
+        set!(simulation, ρ = ρ, uh = uh, w = w, ρe_tot = ρe_tot)
+        run!(simulation)
+        u_end = simulation.integrator.u.nhm
+
+        uh_phy = Geometry.transform.(Ref(Geometry.UVAxis()), u_end.uₕ)
+        w_phy = Geometry.transform.(Ref(Geometry.WAxis()), u_end.w)
+
+        @test maximum(abs.(uh_phy.components.data.:1)) ≤ 1e-11
+        @test maximum(abs.(uh_phy.components.data.:2)) ≤ 1e-11
+        @test maximum(abs.(w_phy |> parent)) ≤ 1.0
```

Comment:
Does this mean, we are satisfied with less than 1m/s on w? Seems large, no?

Reply:
Yes. It is large for the first several hours. It damps away for a longer run (~1e-3 after 1 simulation day).

Reply:
Initial conditions not in discrete balance is a potential cause of the flow.